src/fdk_rdf_parser/organizations/utils.py

```diff
@@ -21,7 +21,8 @@ def org_path_url(org: str) -> str:
 
 
 def organization_number_from_uri(uri: str) -> Optional[str]:
-    match = re.compile(
-        "https://data.brreg.no/enhetsregisteret/api/enheter/(\\d{9})$"
-    ).match(uri)
+    match_uri_0 = "https://data.brreg.no/enhetsregisteret/api/enheter/"
+    match_uri_1 = base_org_url + "/organizations/"
+    match_str = "^(?:" + match_uri_0 + "|" + match_uri_1 + ")(\\d{9})$"
```

Comment:
Hvis ikke jeg tar feil, går det an å bruke r"___" syntaks for å slippe \\ i regex string